addons/account/models/company.py

```diff
@@ -204,6 +208,19 @@ def _check_fiscalyear_last_day(self):
             if rec.fiscalyear_last_day > max_day:
                 raise ValidationError(_("Invalid fiscal year last day"))
 
+    @api.depends("fiscal_position_ids.foreign_vat")
+    def _compute_is_multi_vat_company(self):
+        company_to_foreign_vat_country = {
+            val["company_id"][0]: val["country_ids"]
+            for val in self.env["account.fiscal.position"].read_group(
+                domain=[("company_id", "in", self.ids), ("foreign_vat", "!=", False)],
+                fields=["country_ids:count(country_id)"],
```

Comment:
Probably over optimize but it would be nice to not count all of them but to just check if there is at least one value.